index.html

```diff
@@ -570,7 +570,7 @@ <h2 class="about-heading modal-heading"></h2>
                 <input
                   type="email"
                   name="user-email"
-                  id="username"
+                  id="email-id"
                   class="margin-gap"
```

Comment:
The form can be submitted without an email , kindly fix that and make sure it only accepts emails with small case letters